spec/enumerables_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../enumerables.rb'
+
+# Tests
+RSpec.describe Enumerable do
+  let(:arr) { [1, 2, 3, 4, 5] }
+  let(:my_hash) { { one: 1, two: 2, three: 3, four: 4 } }
+
+  describe 'my_select' do
+    it 'same as select with array' do
+      expect(arr.my_select(&:odd?)).to eql(arr.select(&:odd?))
+    end
```

Comment:
**Please note this for subsequent changes**  This is not the way to test for a result, you guys are basically testing that a===a this would always be true. instead of this   
```expect(arr.my_select(&:odd?)).to eql(arr.select(&:odd?))```
 do something like 
```expect(arr.my_select(&:odd?)).to eql([1,3,5])```
You have to conceptualize the outcome and use it. Please do  this for all other test cases

Reply:
Hi team, thanks for notifying me, yes I misread that, it is a good enough test pattern, please fix other things, I am so sorry about that :pray: . And with regards to testing other methods, basically, Microverse wants you to test every method you previously created.